test/test_results.py

```diff
@@ -14,7 +14,7 @@
 from asv import results, runner, util
 import pytest
 
-from .tools import example_results
+from .tools import example_results # noqa :F401 imported but unused
```

Comment:
For the comment, we don't want the flake8 error, which doesn't add much value. What we want is to explain why we are ignoring the flake8 error. Reading the comment, the obvious thing to think is that this import should be deleted, since it's imported but unused. A helpful comment would be to say that this is required even if unused, so the example_results fixture is loaded.

For this particular case, what it may make more sense is to move the fixtures from `tools.py` to the standard `conftest.py`. This way this import won't be needed, as pytest will take care of everything. We don't want to have such a change in a PR with minor style changes. So, one option would be to:
- Change this line to something like `from . import tools  # noqa F401 needed to load fixtures (see #1234)`
- Remove the other `noqa` because of redefinition of `example_results`, which shouldn't be needed anymore
- Open an issue to explain what's going on here, to move the fixtures to the right location

Reply:
Thank you @datapythonista
for your review let me incorporate these changes

Reply:
Hello @datapythonista this still happens when i remove the # noqa

```
test/test_results.py:165:31: F811 redefinition of unused 'example_results' from line 17
test/test_results.py:208:39: F811 redefinition of unused 'example_results' from line 17
```

Reply:
I left a new review explaining how I would fix this. You need the noqa here with the proposed import, but not the errors you have in your comment I think.